aten/src/ATen/core/boxing/KernelFunction.h

```diff
@@ -86,6 +86,14 @@ class CAFFE2_API KernelFunction final {
   template<class Return, class... Args>
   Return call(const OperatorHandle& opHandle, Args... args) const;
 
+
+  /**
+   * Like call, but with a stack of inputs already passed
```

Comment:
This needs a more detailed description of the calling convention. Does the `stack` get mutated to have the updated return arguments unconditionally, or does it depend on if you took the boxed or unboxed call path?

Reply:
I'll expand the comment; it's for the boxed arguments and will be used if the boxed path is chosen